Src/WitsmlExplorer.Frontend/models/jobs/wellReference.ts

```diff
@@ -0,0 +1,4 @@
+export default interface WellboreReference {
```

Comment:
I assume it was supposed to be WellReference instead of WellboreReference?

Reply:
Correct.